pages/linux/losetup.md

```diff
@@ -0,0 +1,19 @@
+# losetup
+
+> Set up and control loop devices.
+
+- List loop devices with detailed info:
+
+`losetup -a`
+
+- Associate / Attach a device or file to a {{loop}} device:
+
+`sudo losetup /dev/{{loop}} /{{path/to/<file>}}`
```

Comment:
This deviates slightly from how we use tokens. I'd suggest something along these lines:

    - Attach a file to a given loop device:
    
    `sudo losetup /dev/{{loop_device}} /{{path/to/file}}`